packages/nextjs/components/scaffold-eth/Balance.tsx

```diff
@@ -28,6 +28,10 @@ export const Balance = ({ address, className = "", usdMode }: BalanceProps) => {
 
   const [displayUsdMode, setDisplayUsdMode] = useState(price > 0 ? Boolean(usdMode) : false);
```

Comment:
If the price is not loaded in the first render, maybe you can just use here _useState(false)_

Reply:
The price is not loaded in the first component render **+ first-page render**, but the price is already loaded on the store when you navigate through pages... so I think it's useful to keep it to avoid flickering in some situations.